README.md

```diff
@@ -87,7 +87,16 @@ hipsolverSgetrf(hipsolverHandle_t handle,
                 int*              devInfo);
 ```
 
-## Supported Fuctionality
+## Special Considerations with the rocSOLVER Backend
+Due to differences in implementation and API design between rocSOLVER and cuSOLVER, there are special considerations that should be taken into account when using hipSOLVER with the rocSOLVER backend.
+
+While many hipSOLVER functions (modeled after cuSOLVER functions) take a workspace pointer and size as arguments, rocSOLVER maintains its own internal device workspace by default. In order to take advantage of this feature, users may pass a null pointer for the `work` argument of any function when using the rocSOLVER backend, and the workspace will be automatically managed behind-the-scenes.
+
+Note that several functions - namely gesvd, getrs, and potrfBatched - will always use rocSOLVER's internal device workspace management. This may cause performance issues if combined with function calls that receive non-null `work` pointers, as the internal workspace will flip-flop between the user-provided and automatically allocated workspaces. It is recommended for programs that call gesvd, getrs, or potrfBatched to pass null pointers to the `work` arguments of all other functions.
```

Comment:
We may add more functions that do not receive `work` pointer. So, I will re-phrase these paragraph to something like this:

"Note that some functions that do not get a `work` array as an argument, will always use rocSOLVER's internal device workspace management (gesvd, getrs, and potrfBatched, for instance). This may cause performance issues if combined with function calls that receive non-null `work` pointers, as the internal workspace will flip-flop between the user-provided and automatically allocated workspaces. In these cases, it is recommended to always pass null pointers to the `work` arguments for the functions that need it."